lib/server.js

```diff
@@ -45,12 +45,12 @@ var server = function(lingon, ip, port) {
 
     // If we don't have a cached path, or if it's no longer valid:
     // let's find it by searching all the possible filenames
-    if(!pathCache[requestFilePath] || !testPath(pathCache[requestFilePath])) {
+    if(!pathCache["CACHE_" + requestFilePath] || !testPath(pathCache["CACHE_" + requestFilePath])) {
```

Comment:
Yeah, we should add a linter step to catch these..
On Feb 3, 2015 5:57 PM, "Philip von Bargen" <notifications@github.com>
wrote:

> In lib/server.js
> <https://github.com/spotify/lingon/pull/123#discussion_r24019131>:
>
> > @@ -45,12 +45,12 @@ var server = function(lingon, ip, port) {
> >
> >      // If we don't have a cached path, or if it's no longer valid:
> >      // let's find it by searching all the possible filenames
> > -    if(!pathCache[requestFilePath] || !testPath(pathCache[requestFilePath])) {
> > +    if(!pathCache["CACHE_" + requestFilePath] || !testPath(pathCache["CACHE_" + requestFilePath])) {
>
> Don't we have single quotes everywhere else?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/lingon/pull/123/files#r24019131>.
>